package.json

```diff
@@ -4,11 +4,17 @@
   "description": "An ESLint rule to encourage using custom hook abstractions",
   "main": "index.js",
   "files": [
-    "*"
+    "index.js",
+    "rules/*.js",
+    "src/*.js"
   ],
   "scripts": {
-    "test": "jest",
-    "test:watch": "jest --watch"
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test:watch": "node --experimental-vm-modules node_modules/jest/bin/jest.js --watch"
```

Comment:
Can you help me understand the need for the `--experimental-vm-modules` flag here?

Reply:
Jest won't run ESM without this. This flag enables Node.js's experimental VM modules feature, which Jest relies on for ESM support.

In the future, would like to move on to using [Vitest](https://vitest.dev/guide/comparisons#jest)